projects/Mallard/src/components/front/items/helpers/item-tappable.tsx

```diff
@@ -12,7 +12,10 @@ import {
 } from 'react-native';
 import type { CAPIArticle, Issue, ItemSizes } from 'src/common';
 import { ariaHidden } from 'src/helpers/a11y';
-import type { MainStackParamList } from 'src/navigation/NavigationModels';
+import type {
+	CompositeNavigationStackProps,
```

Comment:
Think we can remove this now by the looks of it.

Reply:
Will remove 👍